src/arrayMethodJoin.js

```diff
@@ -4,8 +4,18 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
```

Comment:
Yippee Ki-Yay, you invented great, and I thought to 'fingers to the bones' what to do here. Cool